fontMetricsData.js

```diff
@@ -2097,7 +2097,7 @@ export default {
         "125": [0.08333, 0.69444, 0, 0, 0.525],
         "126": [0, 0.61111, 0, 0, 0.525],
         "127": [0, 0.61111, 0, 0, 0.525],
-        "160": [0, 0, 0, 0, 0.525],
+        "160": [0, 0, 0, 0, 0.25],
```

Comment:
These seem to have reverted, and needs to be fixed. Did you not rerun metric generation after merging or something?